src/Sulu/Component/Rest/ListBuilder/AbstractListBuilder.php

```diff
@@ -336,4 +288,14 @@ public function addGroupBy(AbstractFieldDescriptor $fieldDescriptor)
     {
         $this->groupByFields[$fieldDescriptor->getName()] = $fieldDescriptor;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addExpression(ExpressionInterface $expression)
+    {
+        if ($expression) {
```

Comment:
I guess this check is not needed, because of the `ExpressionInterface` typehint.